과제는 기한내에 했고, push까지 기한 내에 했습니다..
그런데, 결과가 나온 후, key를 지우고 push를 했는데, 이후 실수로 실행을 해버려 key가 없는 상태로 코드가 실행되었고, error메세지가 뜨며 ipynb파일에 변화가 생겼습니다.
이후 오늘 다른 과제를 한 뒤 push를 하는 과정에서 이 과제까지 커밋에 들어가 버렸고, 마치 제출이 늦은것 처럼 되었습니다..
https://github.com/happygeon/Ybigta_assignment/blob/8bc56b9ec52b7c708cdf68a86e7ffd98edc9764a/0111%20REST%20API/240111_REST_API_%EA%B3%BC%EC%A0%9C.ipynb
여기 참고해주시길 바라겠습니다
위 링크는 19일 기준 3일전 제출했을 당시의 ipynb파일 입니다.